### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by vimwitch**

### CODE REVIEW

File: packages/frontend/src/Home.jsx [hunk 1]
```diff
@@ -76,28 +74,26 @@ export default observer(() => {
                 onChange={(event) => { setNonce(event.target.value ?? 0) } }
               ></input>
               <p>current epoch key nonce: {nonce}</p>
-              
+
               <Button onClick={()=> {
-                userContext.requestReputation(posRep, negRep, graffitiPreimage, nonce)
-                setTotalPosRep(Number(posRep)+ Number(totalPosRep))
-                setTotalNegRep(Number(negRep)+ Number(totalNegRep))
```
Comment (vivianjeng) [inline packages/frontend/src/Home.jsx:83]: Here I mean to show the current epoch reputation (but not received)
and after transition, the user can check if all reputation are received in the new epoch

so current rep = current epoch rep (not received) + previous epoch rep (received)
the rep which is received is shown in 

https://github.com/Unirep/create-unirep-app/blob/9a677b4000897cd04772e4d5d2966de1d66af3e6/packages/frontend/src/Home.jsx#L43

Comment (vivianjeng) [reply packages/frontend/src/Home.jsx:83]: or we need a better UI to let users know the rep what they request in the epoch will be processed next epoch.

File: packages/frontend/src/contexts/User.js [hunk 2]
```diff
@@ -42,33 +45,38 @@ class User {
...
+  //       this.latestTransitionedEpoch = currentEpoch
+  //     } catch (err) {
+  //       await new Promise(r => setTimeout(r, 10000))
+  //       continue
+  //     }
+  //     const time = this.userState.calcEpochRemainingTime()
+  //     await new Promise(r => setTimeout(r, time * 1000))
+  //   }
+  // }
+
```
Comment (vivianjeng) [inline packages/frontend/src/contexts/User.js:79]: we can remove this

**Review by vivianjeng — CHANGES_REQUESTED** [review]
This should be changed to `userContext.reputation.posRep` as well

https://github.com/Unirep/create-unirep-app/blob/9a677b4000897cd04772e4d5d2966de1d66af3e6/packages/frontend/src/Home.jsx#L43

**Comment by vimwitch** [comment]
Right I took the auto user state transition code out, but the problem is this repo is intended to help people build attesters, not to be a product. It should be added back in as an optional thing.

File: packages/frontend/src/Home.jsx [hunk 3]
```diff
@@ -42,31 +40,40 @@ export default observer(() => {
...
+              </div>
+              <div>Pos Rep {userContext.posRep?.toString()}</div>
+              <div>Neg Rep {userContext.negRep?.toString()}</div>
+              <div>Graffiti {userContext.graffiti?.toString()}</div>
+              <div style={{ fontSize: '18px', fontWeight: '500'}}>
+                Provable Reputation
+              </div>
+              <div>Pos Rep {userContext.posRep?.toString()}</div>
+              <div>Neg Rep {userContext.negRep?.toString()}</div>
+              <div>Graffiti {userContext.graffiti?.toString()}</div>
```
Comment (vivianjeng) [inline packages/frontend/src/Home.jsx:54]:
```jsx
<div>Pos Rep {userContext.provableReputation.posRep?.toString()}</div> 
<div>Neg Rep {userContext.provableReputation.negRep?.toString()}</div> 
<div>Graffiti {userContext.provableReputation.graffiti?.toString()}</div>
```

**Review by vivianjeng — CHANGES_REQUESTED** [review]

**Review by vivianjeng — APPROVED** [review]
LGTM

**Pull request merged**